microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -1,62 +1,166 @@
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 05/06/2021
+ms.date: 02/16/2022
 ---
 # Manage the user data folder
 
-WebView2 applications interact with user data folders to store browser data, such as cookies, permissions, and cached resources.  Each instance of a WebView2 control is associated with a user data folder.  Each user data folder is unique to a user.
+<!--
+lexicon:
+the "user data folder" is used to store x, y, & z.
+The "user data folder location" is where the user data folder is located; its path.
+Sometimes write "the location of the user data folder", when you want to clearly focus on the location.
+-->
+
+As a WebView2 developer, you need to know about the user data folder for WebView2 apps, including how it's created, how it's used, and how you need to handle it.
+
+WebView2 apps use user data folders to store browser data, such as cookies, permissions, and cached resources.
+*  Each instance of a WebView2 control is associated with a user data folder.
+*  Each user data folder is unique to a user.
 
 
 <!-- ====================================================================== -->
-## Best practices
+## How to create user data folders
+
+By default, WebView2 creates a user data folder in the default location.  The default location works well for most apps.  If your app has specific needs, you can specify a different location.  Make sure that user data folder location has appropriate Read/Write permissions for the WebView2 app runtime.
+
+To specify the location of the user data folder, include the `userDataFolder` parameter when calling [ICoreWebView2Environment](/microsoft-edge/webview2/reference/win32/icorewebview2environment) (Win32) or [CoreWebView2Environment](/dotnet/api/microsoft.web.webview2.core.corewebview2environment) (.NET).  After creation, browser data from your WebView2 control is stored in a subfolder of `userDataFolder`.
 
-User data folders are created automatically by WebView2.  WebView2 developers control the lifetime of the user data folder.  If your application re-uses user data from application sessions, consider saving the user data folders, otherwise you can delete them.  Consider the following scenarios when deciding how to manage your user data folders:
+When `userDataFolder` isn't specified, WebView2 creates user data folders at default locations as follows:
 
-*  If the same user uses your application repeatedly, and the web content of the application relies on the user's data, save the user data folder.
+*  For packaged Windows Store apps, the default user folder is the `ApplicationData\LocalFolder` subfolder in the package's folder.
 
-*  If multiple users use your application repeatedly, create a new user data folder for each new user, and save the user data folder of each user.
+*  For existing desktop apps, the default user data folder is the executable (`exe`) path of your app + `.WebView2`.
```

Comment:
Is this supposed to be /webview2? I dont know if the exe path DOT webview2 makes sense?